code/modules/mob/living/simple_animal/hostile/viscerator.dm

```diff
@@ -38,6 +38,11 @@
 	qdel (src)
 	return
 
+/mob/living/simple_animal/hostile/viscerator/CanAttack(var/atom/the_target)
+	if(isnukeop(the_target) && faction == "syndicate")
```

Comment:
Compile failed because you didn't typecast the target as human before using the define that just jumps straight into whether or not `the_target`'s mind says its a nukeop